internal/config/authentication.go

```diff
@@ -340,8 +357,17 @@ func (a AuthenticationMethodOIDCConfig) info() AuthenticationMethodInfo {
 		}
 	}
 
+	ematchesSplit := strings.Split(a.EmailMatches, ",")
+
+	for _, e := range ematchesSplit {
+		emailMatches = append(emailMatches, e)
+	}
+
 	metadata["providers"] = providers
+	metadata["email_matches"] = emailMatches
```

Comment:
im wondering if this is somehow a security risk.. trying to remember if this endpoint is protected by auth or not

Reply:
It is protected by auth. But without authz it is a bit leaky. Perhaps we should omit this until someone asks for it.

Reply:
We can omit this. I put this in the metadata to validate against initially, but I'll take a different approach.